### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by watchmexiang** [fyangneil/pavement-crack-detection#4]
Hello, I used the original FCN to experiment on your dataset and found that the results obtained by FCN are better than the results of your model under the MIOU indicator. After reading your article, I found that you used The indicator is the indicator of edge detection. You have come to the conclusion that your model is better than FCN. Have you compared the test results of the two models under the MIOU indicator? Thank you.

**Comment by fyangneil** [comment]
We did not use MIoU as a measurement to evaluate the methods. When you use FCN, what is the output? The output layer is 'softmax' or 'sigmoid' ? In our implementation, the output of FCN is 'sigmoid'. When you compute the MIoU, did you take the background into account? In our case, the background should not be considered. In addition, instead of using MIoU, we use AIU (average intersection over union) as a metric to evaluate the compared methods.  For the specific definition of AIU, you are referred to the paper.

**Comment by zhangyuqiqiyuzhang** [comment]
Thanks for your reply, my output layer is softmax. I included the background when comparing MIOU, but compared the experimental results shared with you in the case of background, I found that fcn is better than your result. Is it because of the background? For the sake of this, I am still making further comparisons.

**Comment by zhangyuqiqiyuzhang** [comment]
Thanks for your reply, my output layer is softmax. I included the background when comparing MIOU, but compared the experimental results shared with you in the case of background, I found that fcn is better than your result. Is it because of the background? For the sake of this, I am still making further comparisons.

**Comment by zhangyuqiqiyuzhang** [comment]
Or can you share the following code for calculating both ODS and OIS indicators?

**Comment by fyangneil** [comment]
For the code to compute ODS and OIS, you are referred to
https://github.com/s9xie/hed.
BTW, if you use softmax output, the predicted crack map is a binary map,
but sigmoid gives you a probability map. This may cause some problems when
using MioU metric. For example, how to binarize the probability map? what
kind of threshold should be used to fairly evaluate the predicted results.

Fan

**Comment by fyangneil** [comment]
@zhangyuqiqiyuzhang 
The evaluation tool is provided, please check it out.

**Comment by watchmexiang** [comment]
Thank you for your continued help, can you share the following experimental results of crackforest in each data set, thank you again for your help.

**Comment by zhangyuqiqiyuzhang** [comment]
Or you can send it to my email address, 12647552202@qq.com, thanks again

**Comment by watchmexiang** [comment]
Or you can send it to my email address, 1264754202@qq.com, thanks again

**Comment by watchmexiang** [comment]
I am really sorry to bother you again. I tried to use CrackForest to run some data sets you used to get the test results, but I can't get your result map. It may be my operation problem, so if you are convenient, can you Share the results of Crackforest's experiments in the dataset. If it is not convenient, can you give me some help? Thank you again for your help

**Comment by fyangneil** [comment]
i am a little busy recently. I will share you the result in this weekend,
if i have it.

Fan

**Comment by fyangneil** [comment]
@zhangyuqiqiyuzhang 
I  have uploaded it to google drive, please check it out.

**Comment by watchmexiang** [comment]
I have successfully downloaded the test results, thank you again for your help.

**Comment by fightingchun** [comment]
I'd like to ask you  the difference between traincrop and traindata in the Crack500,thanks!

**Comment by fyangneil** [comment]
the image in train data is raw image.
Images in traincrop are cropped patches on raw image

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
